torchao/quantization/README.md

```diff
@@ -121,6 +132,15 @@ from torchao.quantization.quant_api import change_linear_weights_to_int8_dqtenso
 change_linear_weights_to_int8_dqtensors(model)
 ```
 
+#### A8W8 Float8 Dynamic Quantization
```

Comment:
other fp8 features are added as prototype right now: https://github.com/pytorch/ao/blob/main/torchao/quantization/README.md#prototype-a16w8-float8-weightonly-quantization

I'm not sure if benchmarking is the only blocker to move to normal features, but if it is, then we should move everything under normal features.

cc @vkuzo @drisspg checking if there is anything else we need to do before moving fp8 to be a released feature

Reply:
Yeah for inference I think it is safe to move, lets do that in a follow up PR
cc @jainapurva